frontend/src/bundles/common/components/side-bar/side-bar.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import {
+    Box,
+    Flex,
+    Icon,
+    IconButton,
+    IconMap,
+    Spacer,
+} from '~/bundles/common/components/components.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { Size } from '../icon/icon.js';
+import {
+    CollapseButton,
+    CollapseButtonLink,
+} from './libs/components/components.js';
+
+const SideBar = (): JSX.Element => {
+    const [isCollapsed, setIsCollapsed] = useState(false);
+    const { pathname } = useLocation();
+    const navigate = useNavigate();
+
+    const toggle = useCallback(
```

Comment:
```suggestion
    const handleToggle = useCallback(
```
A10
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md